recipes/qwt/all/conanfile.py

```diff
@@ -65,7 +60,9 @@ def requirements(self):
     def build_requirements(self):
         if self.settings.os == "Windows" and self.settings.compiler == "Visual Studio":
```

Comment:
```suggestion
        if is_msvc(self):
```

and add `from conan.tools.microsoft import is_msvc`